content/en/monitors/incident_management.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Incident Management
+kind: documentation
+disable_toc: true
+description: Create and manage incidents
+---
+
+{{< img src="monitors/incidents/incidents-top-1.png" alt="Incident Management"  style="width:80%;">}}
+
+## Overview
+
+Any event that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these events. Datadog’s Incident Management feature provides a system through which your organization can effectively identify and mitigate incidents. 
+
+Incidents live in Datadog alongside the metrics, traces, and logs you are collecting. You can view and filter incidents that are relevant to you.
+
+In the Datadog paradigm, any of the following are appropriate situations for declaring an incident:
+
+* An issue is or may be impacting customers or services
+* You do not know whether you should call an incident. Notify other people and increase severity appropriately.
+
+<div class="alert alert-warning">
+Incident Management is currently in beta and available to all customers. For more information, reach out to sales@datadoghq.com.
+</div>
+
+## Usage
+
+Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page for a feed of all ongoing incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].
+
+### Creating an incident
+
+You can declare an incident directly from a graph. You can also go to the upper right hand side of the Incidents page and click on “New Incident”. The following dialog box appears.
+
+{{< img src="monitors/incidents/create-1-1.png" alt="New Incident"  style="width:80%;">}}
+
+**Severity**: Denotes the severity of your incident, from SEV-1 (most severe) to SEV-5 (least severe). If the severity of your incident is unknown, select UNKNOWN.
```

Comment:
If your incident is under initial investigation and you do not know the severity yet, select UNKNOWN.

Reply:
^ instead of "If the severity of your incident is unknown, select UNKNOWN."